### PR TITLE
Fix mine count variable names

### PR DESCRIPTION
**Pull request by ThePhilgrim**
This addresses issue #33

### CODE REVIEW

File: minesweeper.py [hunk 1]
```diff
@@ -349,4 +346,5 @@ def difficulty_slider_callback(*args):
 
 new_game()
 root.title("Minesweeper – by Arrinao, The Philgrim, and Master Akuli")
+root.iconphoto(False, tkinter.PhotoImage(file=where_this_file_is / "bomb.png"))
```
Comment (Akuli) [inline minesweeper.py]: Please delete this change from this PR because it's already a separate PR.

**Review by Akuli — CHANGES_REQUESTED** [review]

File: minesweeper.py (hunk 1 above)
Comment (Akuli) [reply minesweeper.py]: I think this should work:

```
git checkout fix_mine_count_variable_names
git revert b00fc89
git push
```

Comment (ThePhilgrim) [reply minesweeper.py]: Hmm, how is it possible that this became a change even in this or? I added it in the other one, pushed, made a new branch and did the changes intended for this or, and pushed those

Comment (Akuli) [reply minesweeper.py]: You probably made the branch for this PR based on the wrong branch. Run `git checkout main` before making the branch for a new PR.

Comment (Akuli) [reply minesweeper.py]: Actually, I think I might be able to fix this for you

Comment (Akuli) [reply minesweeper.py]: Done. Now we just need Arrinao's review

**Review by Akuli — APPROVED** [review]

**Pull request merged**